.github/workflows/CI-try-runtime.yml

```diff
@@ -9,14 +9,14 @@ on:
         description: 'Whether to execute try-runtime step or not'
         required: false
         type: string
-        default: 'yes'
+        default: 'no'
```

Comment:
If default is `no` now you may want to set it to `yes` under orchestrator workflow?

```
  try-runtime-job:
    name: Try runtime job
    uses: ./.github/workflows/CI-try-runtime.yml
```

Reply:
Yes this is intended; this step will fail until the RPC node is connected to the relay chain.
The plan is to switch it back to `yes` again once that will be done.

Reply:
If this is temp you can use the switch when calling the workflow vs changing its default config. For example 
```
uses: ./.github/workflows/CI-try-runtime.yml
    with:
      execute-try-runtime: 'no'
```